.github/workflows/build-and-push-dev-images.yml

```
@@ -28,13 +28,13 @@ on:
         required: false
       lang:
         description: List of languages to build
-        default: 'gcc-toolset, golang, nginx, nodejs, php, python, ruby'
+        default: 'gcc-toolset, golang, nginx, nodejs, php, python, ruby, redis'
```

It's a detail, but we try to respect alphabetical order...
```suggestion
        default: 'gcc-toolset, golang, nginx, nodejs, php, python, redis, ruby'
```